### ISSUE TITLE
Sometimes the site's CSS doesn't load

### ISSUE THREAD

**Issue by rakleed** [2factorauth/frontend#4]
Sometimes the site's CSS doesn't load, and I see errors like this in the console:

`Error with Permissions-Policy header: Parse of permissions policy failed because of errors reported by structured header parser.`

`beta.2fa.directory/:1 Failed to find a valid digest in the 'integrity' attribute for resource 'https://beta.2fa.directory/assets/css/tables.3562fd21a6c14bc772ceb4d53eb1f95d79068f7b6babda6e3cf01f7c4f1343ba113826a8eea26511ecf85820f073110007d1d582b82ee93befeeaeafa64545dc.css' with computed SHA-512 integrity 'prBrGcsMp6yL989wg59I9SAIZkTtNmg3sYoj7N5fA+F2tiREtbGxt2jcdMnW/7xmD7LXbsGeog8IR6+KnaFlpw=='. The resource has been blocked.`

**Comment by Carlgo11** [comment]
That's odd. I had the same issue a few days ago but thought I fixed it by disabling some caching features with Cloudflare that apparently changed content.
If you remove your browser cache does the issue disappear?

**Comment by rakleed** [comment]
I tried clearing the cache for `2fa.directory` and `beta.2fa.directory` and that worked at first. But when I opened the `/about` page, again the styles didn't load. Then on the main page I did a [hard refresh](https://fabricdigital.co.nz/blog/how-to-hard-refresh-your-browser-and-clear-cache), and now all the styles are not loaded here either.

I also tried creating a new profile in Chrome, but the styles didn't load the first time I opened the main page.

**Comment by Carlgo11** [comment]
\*sigh*
Guess I'll have to disable SRI hashes then :disappointed:

**Comment by Carlgo11** [comment]
@rakleed does it work now? (Might have to do a hard refresh `ctrl+shift+R`)

**Comment by rakleed** [comment]
Yes, everything is working correctly now. But if you still want to use SRI, these tips might be helpful: https://stackoverflow.com/questions/65040931/hugo-failed-to-find-a-valid-digest-in-the-integrity-attribute-for-resource

**Comment by rakleed** [comment]
Or are you just trying to do it?

> I had the same issue a few days ago but thought I fixed it by disabling some caching features with Cloudflare that apparently changed cached content.

**Comment by Carlgo11** [comment]
I didn't know that was a feature :open_mouth:
Thanks!
![](https://user-images.githubusercontent.com/3535780/206590346-f07be43e-7dbd-4e42-87f9-73e3c100a2d7.png)
